public/content/developers/docs/nodes-and-clients/client-diversity/index.md

```diff
@@ -80,6 +80,8 @@ Addressing client diversity requires more than individual users to choose minori
 
 [Prysm](https://docs.prylabs.network/docs/getting-started)
 
+[Grandine](https://github.com/grandinetech/grandine)
+
```

Comment:
Would avoid this listing until Grandine is out of beta with at least a v1 release

```suggestion
```